flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/utils/FlinkContainerTestEnvironment.java

```diff
@@ -140,13 +140,15 @@ public void before() {
                 new GenericContainer<>(getFlinkDockerImageTag())
                         .withCommand("jobmanager")
                         .withNetwork(NETWORK)
+                        .withExtraHost("host.docker.internal", "host-gateway")
```

Comment:
Why do we need this setting?

Reply:
The containers of observer and oblogproxy use the `host` network, and `host.docker.internal` is used to request the host server from inside the flink docker container.